shared/components/Schedule/index.scss

```diff
@@ -114,7 +116,7 @@ $left-column-width: 200px;
     }
 
     @media (min-width: $tablet-portrait-width) {
-      width: calc(100% - 200px);
+      width: auto;
```

Comment:
I think this only works because there's still a `width: calc(100% - 200px);` in ScheduleItem css. Don't you think `margin-left` is better than `calc`?